### PR TITLE
Document how to run example AttentionIsAllYouNeed

### PR DESCRIPTION
**Pull request by Oblynx**

### CODE REVIEW

File: example/AttentionIsAllYouNeed/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Attention Is All You Need
+
+This is an implementation of the seminal paper [Attention is all you need [annotated]](https://nlp.seas.harvard.edu/2018/04/03/attention.html)
```
Comment (chengchingwen) [inline example/AttentionIsAllYouNeed/README.md:3]: Should point to the original paper
```suggestion
This is an implementation of the seminal paper [Attention is all you need](https://arxiv.org/abs/1706.03762)
```

File: example/AttentionIsAllYouNeed/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+that introduced the Transformer architecture.
+The architecture is explained [here](https://medium.com/data-science-in-your-pocket/attention-is-all-you-need-understanding-with-example-c8d074c37767).
```
Comment (chengchingwen) [inline example/AttentionIsAllYouNeed/README.md:5]:
```suggestion
The architecture is explained [here](https://nlp.seas.harvard.edu/2018/04/03/attention.html).
```

**Review by chengchingwen — CHANGES_REQUESTED** [review]

**Comment by chengchingwen** [comment]
Thanks!

**Comment by Oblynx** [comment]
Thank you!

**Pull request merged**